test/suite/wtbackup.py

```diff
@@ -124,17 +128,68 @@ def confirmPathDoesNotExist(self, uri, dir):
             'confirmPathDoesNotExist: URI exists, file name matching \"' +
             uri.split(":")[1] + '\" found')
 
+    #
+    # Copy a file into given directory.
+    #
+    def copy_file(self, file, dir):
+        copy_from = file
+        # If it is log file, prepend the path.
+        if self.logpath and "WiredTigerLog" in file:
+            copy_to = dir + '/' + self.logpath
+        else:
+            copy_to = dir
+        shutil.copy(copy_from, copy_to)
+
+    #
+    # Cursor next wrapper function.
+    #
+    def next(self, cur):
+        ret = cur.next()
+        if ret == 0:
+           return True
+        self.assertEqual(ret, wiredtiger.WT_NOTFOUND)
+        return False
```

Comment:
I steered you wrong on this, I think we don't need this function and code can just say: `while cursor.next() == 0:`.
The reason is that the Python layer normally takes care of any errors from any WiredTiger API.  All errors are converted to exceptions (which would fail the Python test).  And the API understands that cursor.next() and some other functions can legally return `WT_NOTFOUND` without error.  So `cursor.next` can only return `0` or `WT_NOTFOUND`, or not return at all and raise an an exception instead.

Reply:
ahh makes alot of sense, changed back